docs/disconnected-updateservice-operator.md

```diff
@@ -0,0 +1,295 @@
+
+```
+oc adm upgrade
+>  Cluster version is 4.8.22
+
+>  Upgradeable=False
+>    Reason: AdminAckRequired
+>    Message: Kubernetes 1.22 and therefore OpenShift 4.9 remove several APIs which require admin consideration. Please see the knowledge article https://access.redhat.com/articles/6329921 for details and instructions.
+>  warning: Cannot display available updates:
+>    Reason: RemoteFailed
+>    Message: Unable to retrieve available updates: Get "https://api.openshift.com/api/upgrades_info/v1/graph?arch=amd64&channel=stable-4.8&id=6f878e7c-cb81-41de-a954-bb6766548318&version=4.8.22": dial tcp 18.210.64.188:443: connect: network is unreachable
+# ...  old public update service url ...
+
+POLICY_ENGINE_GRAPH_URI="$(oc -n openshift-update-service get updateservice karampok -o jsonpath='{.status.policyEngineURI}/api/upgrades_info/v1/graph')"
+PATCH="{\"spec\":{\"upstream\":\"${POLICY_ENGINE_GRAPH_URI}\"}}"
+oc patch clusterversion version -p $PATCH --type merge
+```
+
+
+### Make client cluster trust UpdateService server certificate
+
+Unless your UpdateService tls server certificate is somehow trusted by the client OCP cluster (e.g. well known root CA), then you probably get the
+following error:
+
+```
+oc adm upgrade
+> Cluster version is 4.8.22
+> Upgradeable=False
+>   Reason: AdminAckRequired
+>   Message: Kubernetes 1.22 and therefore OpenShift 4.9 remove several APIs which require admin consideration. Please see the knowledge article https://access.redhat.com/articles/6329921 for details and instructions.
+> warning: Cannot display available updates:
+>   Reason: RemoteFailed
+>   Message: Unable to retrieve available updates: Get "https://karampok-policy-engine-route-openshift-update-service.apps.bcn.hub-virtual.lab/api/upgrades_info/v1/graph?arch=amd64&channel=stable-4.8&id=6f878e7c-cb81-41de-a954-bb6766548318&version=4.8.22": 
+#  ... x509: certificate signed by unknown authority ....
+```
+
+If the client cluster, is different than the cluster where UpdateService was installed, you can either add the CA cert 
+on the node or in the proxy configuration: 
+
+```
+    oc create configmap trusted-updateservice-ca -n openshift-config --from-file=ca-bundle.crt=OCPRootCA.pem
+    oc patch proxy/cluster -p '{"spec":{"trustedCA":{"name":"trusted-updateservice-ca"}}}' --type merge
+    oc edit proxy/cluster
+    # ... wait until pods are restarting ..
+```
+
+Finally you should get the following:
+
+```
+dan adm upgrade
+> Cluster version is 4.8.22
+> Upgradeable=False
+>  Reason: AdminAckRequired
+>  Message: Kubernetes 1.22 and therefore OpenShift 4.9 remove several APIs which require admin consideration. Please see the knowledge article https://access.redhat.com/articles/6329921 for details and instructions.
+> Updates:
+> VERSION IMAGE
+> 4.8.23  {DISCONNECTED_REGISTRY}/ocp4/release@sha256:3fab205d36c66825423274eac90f4c142a18cdf358b4a666a1783d325afba860
+```
+
+## Update Client Cluster
+
+We can force the update using the CLI
```

Comment:
Lets not mention `force` as the cluster may lose supported status. Needs to be reformatted into `in case of emergency` or smth similar. 
Same for "--allow-upgrade-with-warnings".

"--to-latest" is also probably not a great idea, lets rework into "to desired version" and adding that "--to-latest" is a shortcut for that